core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -62,18 +62,18 @@ abstract contract FeePayer is Testable {
 
     /**
      * @notice Constructs the FeePayer contract. Called by child contracts.
-     * @param collateralAddress ERC20 token that is used as the underlying collateral for the synthetic.
-     * @param finderAddress UMA protocol Finder used to discover other protocol contracts.
-     * @param timerAddress Contract that stores the current time in a testing environment.
+     * @param _collateralAddress ERC20 token that is used as the underlying collateral for the synthetic.
+     * @param _finderAddress UMA protocol Finder used to discover other protocol contracts.
+     * @param _timerAddress Contract that stores the current time in a testing environment.
      * Must be set to 0x0 for production environments that use live time.
      */
     constructor(
-        address collateralAddress,
-        address finderAddress,
-        address timerAddress
-    ) public Testable(timerAddress) {
```

Comment:
This should be part of a larger fix  to precede all constructor arguments with underscores (see STYLE.md), but re-naming `timerAddress` to `_timerAddress` appeases the `state-variable-shadowing` detector because `FeePayer` inherits `Testable` which has a state variable `timerAddress`

Reply:
yup. this is good. I didnt make this change in the previous PR fix for this formatting as this was only for the DVM. I agree we should apply this across the board.